doc/source/contributor-ref-good-first-contributions.rst

```diff
@@ -37,6 +40,4 @@ We wish we had more time to write usage examples because we believe they help
 users to get started with building what they want to build. Here are a few
 ideas where we'd be happy to accept a PR:
```

Comment:
I think this section may not be as relevant anymore. Instead, we can redirect users to contribute to the unofficial examples section on Flower Discuss.